docs/resources/Channel.md

```diff
@@ -848,11 +848,12 @@ Returns a [message](#DOCS_RESOURCES_CHANNEL/message-object) object. Fires a [Mes
 
 ###### JSON Params
 
-| Field   | Type                                                 | Description                                                                                                                             |
-|---------|------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------|
-| content | string                                               | the new message contents (up to 2000 characters)                                                                                        |
-| embed   | [embed](#DOCS_RESOURCES_CHANNEL/embed-object) object | embedded `rich` content                                                                                                                 |
-| flags   | integer                                              | edit the [flags](#DOCS_RESOURCES_CHANNEL/message-object-message-flags) of a message (only `SUPPRESS_EMBEDS` can currently be set/unset) |
+| Field            | Type                                                                      | Description                                                                                                                             |
+|------------------|---------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------|
+| content          | string                                                                    | the new message contents (up to 2000 characters)                                                                                        |
+| embed            | [embed](#DOCS_RESOURCES_CHANNEL/embed-object) object                      | embedded `rich` content                                                                                                                 |
+| flags            | integer                                                                   | edit the [flags](#DOCS_RESOURCES_CHANNEL/message-object-message-flags) of a message (only `SUPPRESS_EMBEDS` can currently be set/unset) |
+| allowed_mentions | [allowed mention object](#DOCS_RESOURCES_CHANNEL/allowed-mentions-object) | allowed mentions for the message                                                                                                        |
```

Comment:
this is already added in #1679

Reply:
That's been open since May, so I don't think it's worth removing it from this PR. If this gets merged first, it'll be a win-win.